### ISSUE TITLE
locale setting for debian

### ISSUE THREAD

**Issue by emfox** [phusion/baseimage-docker#276]
I know it's a ubuntu image, but I'd like to use it for debian too, by replacing 'FROM ubuntu:14.04' to 'FROM debian:jessie' in Dockerfile, everything works except locales while buidling, so I changed it a bit and all ok (of course should alter sources.list too, ignore it).
I'm not very clear if it's a correct way but it do works, could you intergrate this little change?

```
diff --git a/image/prepare.sh b/image/prepare.sh
index f0512d4..b8873b2 100755
--- a/image/prepare.sh
+++ b/image/prepare.sh
@@ -16,8 +16,7 @@ mkdir -p /etc/container_environment
 echo -n no > /etc/container_environment/INITRD

 ## Enable Ubuntu Universe and Multiverse.
-sed -i 's/^#\s*\(deb.*universe\)$/\1/g' /etc/apt/sources.list
-sed -i 's/^#\s*\(deb.*multiverse\)$/\1/g' /etc/apt/sources.list
+echo 'deb http://mirrors.163.com/debian jessie main' > /etc/apt/sources.list
 apt-get update

 ## Fix some issues with APT packages.
@@ -42,7 +41,8 @@ $minimal_apt_get_install software-properties-common
 apt-get dist-upgrade -y --no-install-recommends

 ## Fix locale.
-$minimal_apt_get_install language-pack-en
+$minimal_apt_get_install locales
+echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
 locale-gen en_US
 update-locale LANG=en_US.UTF-8 LC_CTYPE=en_US.UTF-8
 echo -n en_US.UTF-8 > /etc/container_environment/LANG
```

**Comment by rmoriz** [comment]
Please consider using http://httpredir.debian.org/ instead of mirrors.163.com

**Comment by Theaxiom** [comment]
This should now be fixed by #336
